### PR TITLE
Add Tizen support for SkiaSharp

### PR DESCRIPTION
**Pull request by vincent-kwon**
Modifications to enable Tizen support for SkiaSharp Xamarin.Forms for Tizen.

### CODE REVIEW

File: gn/BUILDCONFIG.gn [hunk 1]
```diff
@@ -45,6 +49,7 @@ is_linux = current_os == "linux"
 is_mac = current_os == "mac"
 is_win = current_os == "win" || current_os == "winrt"
 is_winrt = current_os == "winrt"
+is_tizen = building_for_tizen
```
Comment (rookiejava) [inline gn/BUILDCONFIG.gn]: seems always return false

File: gn/BUILDCONFIG.gn [hunk 2]
```diff
@@ -18,6 +18,10 @@ declare_args() {
   }
   linux_soname_version = ""
   sanitize = ""
+
+  building_for_tizen = false
```
Comment (mattleibow) [inline gn/BUILDCONFIG.gn]: Could we replace this with a use of `target_os="tizen"` in the section below? Basically, like what we did with `winrt` and `tvos`.

File: gn/BUILDCONFIG.gn (hunk 1 above)
Comment (mattleibow) [inline gn/BUILDCONFIG.gn]: Maybe do `is_tizen = current_os == "tizen"`. I am assuming that you are using the Linux target, so you can update that to `is_linux = current_os == "linux" || current_os == "tizen"`. There no real reason other than consistency at this point.

File: third_party/freetype2/BUILD.gn [hunk 3]
```diff
@@ -12,9 +12,16 @@ declare_args() {
 import("../third_party.gni")
 
 if (skia_use_system_freetype2) {
-  system("freetype2") {
-    include_dirs = [ "/usr/include/freetype2" ]
-    libs = [ "freetype" ]
+  if (is_tizen) {
+    system("freetype2") {
+      cflags = [ "-I=/usr/include/freetype2" ]
```
Comment (mattleibow) [inline third_party/freetype2/BUILD.gn]: Just checking to see if there was a reason for this specific change for Tizen?

**Review by mattleibow — CHANGES_REQUESTED** [review]
Great PR. Just a small change if possible and a question.

File: gn/BUILDCONFIG.gn (hunk 2 above)
Comment (vincent-kwon) [reply gn/BUILDCONFIG.gn]: Done.

File: gn/BUILDCONFIG.gn (hunk 1 above)
Comment (vincent-kwon) [reply gn/BUILDCONFIG.gn]: Done.

File: third_party/freetype2/BUILD.gn (hunk 3 above)
Comment (vincent-kwon) [reply third_party/freetype2/BUILD.gn]: It was mistakenly left over. Fixed. I missed it. Thanks.

**Review by mattleibow — APPROVED** [review]

**Comment by mattleibow** [comment]
Nice! As soon as we figure out what happened to your CLA, I will merge.

**Pull request merged**